docs/source/usage/usage.rst

```diff
@@ -17,3 +17,7 @@ You can start conpot by just running it from the commandline.
     2013-04-12 16:09:27,143 Modbus traffic from 127.0.0.1: {'request_pdu': '0100010008', 'function_code': 1, 'slave_id': 1, 'response_pdu': '0101ff'} (b763654f-c9d8-45ae-b35a-824dfc220911)
     2013-04-12 16:09:27,144 Client disconnected. (b763654f-c9d8-45ae-b35a-824dfc220911)
 
+If you try to run Conpot with the sudo command:
+box$ sudo conpot
+and you get an error saying command not found, and you CAN run conpot without sudo, but you get permission denied notices then use this command:
+box$ sudo -E env "PATH=$PATH" conpot --template <template name>
```

Comment:
```suggestion
If you run Conpot with the sudo: `$ sudo conpot` and you get a `command not found` error, use then `env` parameter to user your users `PATH` for root:
`sudo -E env "PATH=$PATH" conpot --template <template name>`
```

Reply:
This may be even easier:  
`sudo $(which conpot) --template <template name>`

This works because virtualenv generates small stub scripts that do all the magic for you, as evidenced by running `cat $(which conpot)` inside a virtualenv. (The "magic" is mostly in the shebang.)

Thus `sudo /full/path/to/bin/inside/my/venv` would work even without having the virtualenv activated. No environment trickery required.

The `$(which conpot)` trick of course takes advantage of the fact that the virtualenv **is** activated at the time and that `which` is executed in the non-`sudo` environment.

Reply:
Sorry, this was assuming this was all about virtualenvs, which is where those errors usually occur. I should have looked at the entire context.